### PR TITLE
Added contents for IBM Security Guardium Insights v3.1.x deployment guide

### PR DESCRIPTION
**Pull request by shinushaju**
*Description of changes:*

- Updated partner editable contents and added new images
- Updated .gitignore file and added .metadata and CODEOWNERS

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/deployment_guide/partner_editable/overview.adoc [hunk 1]
```diff
@@ -1,5 +1,35 @@
-This Quick Start deploys {partner-product-name} on the AWS Cloud. This guide covers the steps necessary to deploy this Quick Start.
+This Quick Start deploys {partner-product-name} on the AWS Cloud. If you are unfamiliar with AWS Quick Starts, refer to the https://fwd.aws/rA69w?[AWS Quick Start General Information Guide^].
```
Comment (vsnyc) [inline docs/deployment_guide/partner_editable/overview.adoc:1]: Keep the original wording. `If you are unfamiliar ...` sentence comes from the boilerplate language and is getting repeated.

```suggestion
This Quick Start deploys {partner-product-name} on the AWS Cloud. This guide covers the steps necessary to deploy this Quick Start.
```

File: docs/deployment_guide/partner_editable/overview.adoc [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+{partner-product-name} provides new analytics to enhance threat investigations - and it provides quick reporting functionality. Risk scoring and alerting in Guardium
+Insights help you prioritize your activities.
+
+After your data sources are scanned, Guardium Insights provides this information to help you analyze data:
+
+* Data sources and users that are at risk
+* Anomalies that indicate events that are atypical
+* Reports
+* Activity that has been undertaken as a result of data analysis
+
```
Comment (vsnyc) [inline docs/deployment_guide/partner_editable/overview.adoc:20]: This rest of this section is verbatim copied, please remove it and point to your documentation (`https://www.ibm.com/docs/en/guardium-insights/3.1.x?topic=overview`) instead. We want to keep the deployment guide short and refer to your documentation where necessary.

File: docs/deployment_guide/partner_editable/post_deployment.adoc [hunk 3]
```diff
@@ -1,3 +1,64 @@
...
+.Retrieve secret value for console password
+[link=images/retrieve-secret.png]
+image::../docs/deployment_guide/images/retrieve-secret.png[Retrieve secret value for console password]
+
+{empty} +
+Upon logging in to the {partner-product-name} web client URL, the {partner-product-name} welcome page opens, as shown in <<_gi_welcome_page>>.
+
+[#_gi_welcome_page]
+.Welcome page for {partner-product-short-name} web client
+[link=images/gi-admin-welcome-page.png]
```
Comment (vsnyc) [inline docs/deployment_guide/partner_editable/post_deployment.adoc:40]: Should be:
```suggestion
[link=images/gi-welcome-page.png]
```

**Review by vsnyc — CHANGES_REQUESTED** [review]
Please address the changes requested in the deployment guide.

File: docs/deployment_guide/partner_editable/architecture.adoc [hunk 4]
```diff
@@ -11,18 +11,28 @@ image::../docs/deployment_guide/images/architecture_diagram.png[Architecture]
...
-Shell (SSH) access to Amazon Elastic Compute Cloud (Amazon EC2) instances in public and private subnets.*
+** A boot node Amazon EC2 instance that also serves as a bastion host to allow inbound Secure Shell (SSH) access to EC2 instances in the private subnets.
 * In the private subnets:
-** <item>.
-** <item>.
+** Red Hat OpenShift Container Platform (OCP) master nodes in a single Availability Zone.
+** OCP compute nodes with OpenShift autoscaling for hosting the {partner-product-name} capabilities. Amazon EBS disks are mounted on the compute nodes for container-persistent data.
 // Add bullet points for any additional components that are included in the deployment. Ensure that the additional components are shown in the architecture diagram. End each bullet with a period.
-* <describe any additional components>.
+* Classic Load Balancers spanning the public subnets for accessing {partner-product-name} from a web browser.
```
[the remark this answers is not in the document]
Comment (ShinuShaju) [reply docs/deployment_guide/partner_editable/architecture.adoc:25]: 4 Classic Load Balancers will be created during the deployment.

**Pull request closed without merging**